doc/getting_started/installation_linux.rst

```diff
@@ -65,65 +70,77 @@ Install Requirements and Dependencies
 Note that both Ninja and Make are installed with these instructions; you only
 need one.
 
-Ubuntu:
+.. tabs::
 
-.. code-block:: console
+   .. group-tab:: Ubuntu
```

Comment:
This doesn't seem to work? I only see the Ubuntu instructions, but no tabs with other operating systems

Reply:
Where is the Linux specific page in fact, I can't find it? I just see this:

![image](https://user-images.githubusercontent.com/12450381/65236508-0cdbca80-dad9-11e9-843e-f7725ca37fd3.png)

Reply:
Found it under the "Beyond the GSG" section, but as I write there, not very intuitive IMHO